app/components/auk/tab.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@glimmer/component';
+import { computed } from '@ember/object';
+import { assert } from '@ember/debug';
+import { isPresent } from '@ember/utils';
+
+/**
+ * A (nav) tab. Takes most arguments that LinkTo takes.
+ *
+ * @argument {Number} counter: Count number to display next to tab label
+ * @argument {String} layout: can be (default, "icon-left") or "icon-right"
+ * @argument {Boolean} isHierarchicalBack: Flag to apply custom styling for "hierarchical back button"-tab
```

Comment:
Het kan misschien interessant zijn om de functionaliteit rond `isHierarchicalBack` wat uit te breiden zodat deze ook automatisch het correct icon voorziet binnen de tab (en eventuele copy binnen de tab er uit haalt)?

(moet niet perse nu mee genomen worden, maar moet misschien eens besproken worden)

Reply:
+1 op dat vlak

Die isHierarchicalBack als template gebruiken vond ik persoonlijk iets handig.
Dan kan je gewoon `<Auk::Tab @isHierarchicalBack={{true}}>` gebruiken klaar ermee.

Als ik code nu zie, moet je eigenlijk `@icon` meegeven, want het werkt momenteel niet zoals te zien in publicaties

![image](https://user-images.githubusercontent.com/22245223/121900154-652f1300-cd25-11eb-832a-54a60d1af941.png)

Reply:
Een vergissing. Ik pas het aan.